colorbleed/plugins/maya/publish/extract_pointcache.py

```diff
@@ -32,6 +32,12 @@ def process(self, instance):
             start -= handles
             end += handles
 
+        attrs = instance.data.get("attr", "").split(";")
+        if not attrs:
+            attrs = ["cbId"]
```

Comment:
Actually, `cbId` should *always* be included. As such even when `attr` is specified the `cbId` should be appended to the list. I guess it comes down to always doing `attrs += ["cbId"]`